app/controllers/transactions_controller.rb

```diff
@@ -12,8 +22,32 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @transaction.update(transaction_params)
+      redirect_to root_path
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy
+    @transaction.destroy
+  end
```

Comment:
@sinpai tests for `#edit`, `#update`, `#destroy` actions?

Reply:
Will add them